code/modules/modular_computers/_description.dm

```diff
@@ -79,7 +79,7 @@ Research - Requires "Research and Development" access on ID card (ie. RD, Roboti
 	NTNet Administration Console - Allows remote configuration of NTNet Relay - CAUTION: If NTNet is turned off it won't be possible to turn it on again from the computer, as operation requires NTNet to work! Requires extra access: "Research Director"
 	NTNet Monitor - Allows monitoring of NTNet and it's various components, including simplified network logs and system status.
 
-Security - Requires "Security" access on ID card (ie. HOS, Security officer, Inspector)
```

Comment:
is this correct? we use inspector here

Reply:
Its not yet adapted, i only made it compilable